### DISCUSSION TITLE
The screenshot is not found after screenshotting

### DISCUSSION THREAD

**Discussion by nory233** [yasirkula/UnityNativeGallery#249] [Q&A]
Hi , so I have imported this to my project and my screenshot button seems to be fine even when I click it in editor it says that its successfully saving it in gallery  in editor something like that , I have done it for a n android phone when i build and all i click to screen shot it asks if i allow for this app to access the photos , media etc i hit ok , and all good but when i head into my gallery i dont find any screenshots in there ? why and what could be the issue as its weird ? PLEASE HELP :(

**Comment by yasirkula** [comment]
Can you check Logcat logs for an error message?

**Reply by nory233** [reply]
sorry but from where can i check that ? , there arent any errors in general

**Reply by nory233** [reply]
I just dont seem to find the image after the  screenshot

**Reply by yasirkula** [reply]
If there is an error in NativeGallery, the image may not appear in gallery. You can use Unity's Android Logcat package to view the connected device's runtime logs.

**Reply by nory233** [reply]
so what shall i do now import logcat package ? and use it in code instead of native gallery , because it doesnt show errors for native gallery,weird  ,,well it did have errors at the beginning but they  are gone

**Reply by yasirkula** [reply]
https://docs.unity3d.com/Packages/com.unity.mobile.android-logcat@0.1/manual/index.html
